src/lucky/force_ssl_handler.cr

```diff
@@ -46,7 +48,19 @@ class Lucky::ForceSSLHandler
 
   private def redirect_to_secure_version(context : HTTP::Server::Context)
     context.response.status_code = settings.redirect_status
+    add_hsts_if_enabled(context)
     context.response.headers["Location"] =
       "https://#{context.request.host}#{context.request.resource}"
   end
+
+  # Read more about [HSTS](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security)
+  private def add_hsts_if_enabled(context : HTTP::Server::Context)
+    sts_value = settings.hsts.try do |hsts|
+      String.build do |s|
+        s << "max-age=#{hsts[:max_age].total_seconds.to_i}"
+        s << "; includeSubDomains" if hsts[:include_subdomains]
+      end
+    end
+    context.response.headers["Strict-Transport-Security"] = sts_value if sts_value
```

Comment:
I'm pretty sure this can be done inside the try block without asking to a temp variable. Might be a little less code/conditionals that way